pkg/bisect/bisect.go

```diff
@@ -543,7 +551,10 @@ func (env *env) test() (*testResult, error) {
 	results, err := env.inst.Test(numTests, cfg.Repro.Syz, cfg.Repro.Opts, cfg.Repro.C)
 	env.testTime += time.Since(testStart)
 	if err != nil {
-		env.log("failed: %v", err)
+		res.rep = &report.Report{
```

Comment:
I wonder if it makes sense to abort bisection in case of such errors. AFAIK `Test` errors typically mean there's something wrong with the config/environment or e.g. there's a temporary GCE flake. These are not kernel-related errors, so by skipping such commit we can actually break the bisection.

Reply:
I don't think skipping usually breaks bisections. Git will figure out via the later steps whether it can rule out the skipped commits as the cause/fix. If it can't the skipped commit will become part of the list of possible causes/fixes in the dashboard.

Of course this is not completely bullet proof, but:
1) Ranges of kernel history are boot broken. We would need to maintain a list with known bad commits to skip in syzkaller. I think this is not feasible.
2) We don't retry failed bisections and returning an error would abort the bisection.

Reply:
Okay, for now let's leave this as is. But I think that, ultimately, it's still more reasonable to let bisections immediately fail and then restart them e.g. at least a couple of weeks afterwards.